scripts/standardising.py

```diff
@@ -1,80 +1,100 @@
 import os
+from multiprocessing import Pool
 from typing import List
 
 from linz_logger import get_log
 
 from scripts.aws.aws_helper import parse_path
-from scripts.cli.cli_helper import parse_source
+from scripts.cli.cli_helper import is_argo, parse_source
 from scripts.files.files_helper import get_file_name_from_path, is_tiff
 from scripts.gdal.gdal_helper import run_gdal
 from scripts.logging.time_helper import time_in_ms
 
 
-def standardising(files: List[str]) -> List[str]:
+def start_standardising(files: List[str], argo_env: bool) -> List[str]:
```

Comment:
would be good to do something like `cryptocurrency: int` and specify the number of workers to use, as `argo_env` doesn't really make too much sense to the caller.

Reply:
Thanks, could you elaborate further on `cryptocurrency` please?

Reply:
LOL darn autocorrect! `concurrency` 

so then this just takes the number of threads rather than a opaque `is_in_argo`